Add lesson 19 quiz (#54)
ReDI-School/nrw-html-and-css-2021-spring: lesson19/index.md

The goal of this class is to make sure that every student has reached checkpoint 18 in their own portfolio projects. Teachers would help students if they have not reached the checkpoint yet.

> ## 2. Quiz

Reviewer: Could we also add a separate file with all the quiz questions and answers (with explanation if needed) because this way we would save this for next semester as well?

Author: Good idea. I will make a file with the questions, but maybe we should not merge it until after the class :). I will add it in a branch

Reviewer: I think there is no problem if we merge it now. I do not expect people looking around in our GitHub repo. But if they do then it is their responsibility to not cheat. You can't cheat life :D

Author: 😄 👍

Author: Quiz file added with explanations

> 

Reviewer: Could you also provide a little bit of explanation for the answer of question 4. I think I get it but I want to be 100% sure

Author: yea, I can

> During the class, there will be a quiz to review CSS Cascade, Specificity & Inheritance. The questions for the quiz are in [quiz.md](./quiz/quiz.md). A Slido link will be shared in the class to facilitate the quiz. 

Reviewer: ```suggestion
During the class, there will be a quiz to review CSS Cascade, Specificity & Inheritance. The questions for the quiz are in [quiz.md](./quiz/quiz.md) (SPOILER ALERT: Please do not open this link before the quiz is over). A Slido link will be shared in the class to facilitate the quiz. 
```

> During the class, there will be a quiz to review CSS Cascade, Specificity & Inheritance. A Slido link for the quiz will be shared in the class. Images present in the `quiz` folder of this lesson are used in the quiz.

Reviewer: I don't see the image for question 5 and 6?

Author: Questions 4,5 and 6 uses the same image lesson19/quiz/exercise-class-2-quiz-image-3.png

Reviewer: I think the image does not contain B and C

Author: Really? In pink.... A is on the left. B and C on the right. 🧐

Reviewer: Ah yes, I somehow missed that I can horizontally scroll on it. It's there 👍

> During the class, there will be a quiz to review CSS Cascade, Specificity & Inheritance. A Slido link for the quiz will be shared in the class. Images present in the `quiz` folder of this lesson are used in the quiz.

Author: ```suggestion
During the class, there will be a quiz to review CSS Cascade, Specificity & Inheritance. The questions for the quiz are in [quiz.md](./quiz/quiz.md). A Slido link will be shared in the class to facilitate the quiz. 
```

> 

Reviewer: I think the answer for question 9 is wrong. The `.body` class is not applied anywhere. Therefore, the font-family on the p would be the default font-family. I suggest adding another answer `default` and making this correct, because it is a nice catch which also teaches students that classNames are different than html tag style definitions.

Author: Oh. It's a typo. It's meant to be `body` not `.body`... Please review it again that way, and see if the answer then makes sense.

Reviewer: Then it would make sense but I was probably aiming at leaving it `.body` as it is the last question and we could make it a little bit more difficult :D

Author: Oh, that would have been nice, but I already recreated the image before reading your comment. Sorry. I left it as `body`.

Reviewer: Alright 👍
